R/utils-cut.R

```diff
@@ -0,0 +1,229 @@
+# these functions largely borrowed from ggplot2.
+# ive copied rather than imported them for increased control
+# this was decided after finding a bug or two
+
+#' Discretise numeric data into equal ranges
+#'
+#' This function makes `n` groups with equal range
+#'
+#' @param x numeric vector
+#' @param n number of intervals to create, OR
+#' @param length length of each interval
```

Comment:
if i want to cut data into intervals of a specified legnth, i should use cut_width instead, no? If yes, i think we can remove the length param here. But i may have misunderstood!

Reply:
youre probably right, but i wasnt really aiming to clean up cut* or change its api right now without good reason